app/models/track/maintainership.rb

```diff
@@ -0,0 +1,13 @@
+class Track::Maintainership < ApplicationRecord
+  belongs_to :track
+  belongs_to :maintainer,
+    class_name: "User",
+    foreign_key: :user_id,
+    inverse_of: :maintainerships
+
+  enum maintainer_type: { regular: 0, polyglot: 1 }, _suffix: :maintainer
```

Comment:
Let me know what you think about this name. I've added the suffix here as `:regular_maintainer` felt better than `:maintainer_regular`.

Reply:
I've toyed with having `track` and `polyglot` values for the type, as well as making `polyglot` a `bool`. The current setup requires us to add any polyglot maintainers to each track, but I think that's fine.